TOPdeskPS/functions/Send-TdWebDAVFile.ps1

```diff
@@ -86,9 +84,9 @@
                 Method      = 'Put'
                 InFile      = $File
                 Credential  = $Credential
-                ContentType = 'application/x-www-form-urlencoded'
+                ContentType = 'application/octet-stream'
             }
-            Invoke-WebRequest @Params
+            Invoke-WebRequest @Params | Out-Null
```

Comment:
Let's get rid of the out-null because it will prevent us from seeing the status of our web request.  Great work

Reply:
And removed!